### PR TITLE
recruit-cn:update tpe positions

### PR DESCRIPTION
**Pull request by Soline324**
@xuechunL @YiniXu9506 PTAL

### CODE REVIEW

File: content/recruit-cn/engineering/bizdev-qa-engineer.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: TiDB 商业产品开发 - 测试开发工程师
+tags:
+  - Engineering
+location: 北京，上海，广州，杭州，成都，Remote
+avatar: qa
+aliases:
```
Comment (xuechunL) [inline content/recruit-cn/engineering/bizdev-qa-engineer.md]: The `aliases` attribute is not needed here.

File: content/recruit-cn/engineering/bizdev-engineering-efficiency-engineer.md [hunk 2]
```diff
@@ -3,9 +3,7 @@ title: TiDB 商业产品开发 - 工程效率研发工程师
 tags:
   - Engineering
 location: 北京，上海，广州，杭州，成都，Remote
-avatar: efficiency
-aliases:
-  - /recruit-cn/engineer/bizdev-engineering-efficiency-engineer/
+avatar: database
```
Comment (xuechunL) [inline content/recruit-cn/engineering/bizdev-engineering-efficiency-engineer.md]: Replace `avatar: database` to `avatar: engineer`.

File: content/recruit-cn/engineering/bizdev-qa-engineer.md [hunk 3]
```diff
@@ -3,9 +3,7 @@ title: TiDB 商业产品开发 - 测试开发工程师
 tags:
   - Engineering
 location: 北京，上海，广州，杭州，成都，Remote
-avatar: qa
-aliases:
-  - /recruit-cn/engineer/bizdev-qa-engineer/
+avatar: database
```
Comment (xuechunL) [inline content/recruit-cn/engineering/bizdev-qa-engineer.md]: Replace `avatar: database` to `avatar: engineer`.

File: content/recruit-cn/engineering/bizdev-qa-engineer.md (hunk 1 above)
Comment (Soline324) [reply content/recruit-cn/engineering/bizdev-qa-engineer.md]: ok

File: content/recruit-cn/engineering/bizdev-engineering-efficiency-engineer.md (hunk 2 above)
Comment (Soline324) [reply content/recruit-cn/engineering/bizdev-engineering-efficiency-engineer.md]: ok

File: content/recruit-cn/engineering/bizdev-qa-engineer.md (hunk 3 above)
Comment (Soline324) [reply content/recruit-cn/engineering/bizdev-qa-engineer.md]: ok

**Pull request merged**